charts/ccd-definition-store-api/values.preview.template.yaml

```diff
@@ -23,7 +23,7 @@ java:
     ELASTIC_SEARCH_ENABLED: true
 
     # TODO: remove TEMPORARY OVERRIDE FOR GlobalSearch R2, reinstate: ELASTIC_SEARCH_HOST: ccd-data-store-api-pr-1260-es-master
-    ELASTIC_SEARCH_HOST: ccd-data-store-api-pr-1836-es-master
+    ELASTIC_SEARCH_HOST: ccd-data-store-api-pr-1828-es-master
```

Comment:
Temporary values should be updated prior to merge into `Global_Search_Release2` branch and then removed before that branch is merged for release.